src/lwc/geTemplateBuilderFormField/geTemplateBuilderFormField.js

```diff
@@ -54,6 +54,16 @@ export default class geTemplateBuilderFormField extends LightningElement {
         return null;
     }
 
+    get fieldApiName() {
```

Comment:
Just curious how we got here with field having either "apiName" or "fieldApiName" property?

Reply:
Yeah, this bothered me as well. This is because when we get the describe info from `getObjectInfo` uiObjectInfoApi we get back the entire object describe along with all the fields. The field api name key from that is `apiName`. When we do `import ... from '@salesforce/schema/...` the field api name key is `fieldApiName`.

Reply:
Ugh, bummer. But that makes sense why we would need to handle both cases!